tests/behaviour/inspectcli.feature

```diff
@@ -750,4 +750,61 @@ Feature: Behaviour testing of csvcubed inspect.
 
     Scenario: inspect should output error when the metadata json-ld input does not exist
         Given a none existing test-case file "cli/inspect/not_exists.csv-metadata.json"
-        Then the file not found error is displayed "Could not find test-case file"
\ No newline at end of file
+        Then the file not found error is displayed "Could not find test-case file"
+
+    Scenario: inspect should produce the expected printable for code list metadata json-ld with concepts that uses uri safe identifier override.
```

Comment:
The purpose of the test is to check that the newer-style code-list outputs from csvcubed can be inspected properly. It isn't specifically the URI safe identifer override changes we need to test here.

Reply:
The two tests are now added.